src/bin/log/tracing_chrome.rs

```diff
@@ -0,0 +1,625 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2020 Thoren Paulson
+//! This file is taken unmodified from the following link, except for file attributes and
+//! `extern crate` at the top. It is licensed under MIT.
```

Comment:
```suggestion
//! `extern crate` at the top.
```
This is now redundant.